libraries/lib-numeric-formats/formatters/BeatsNumericConverterFormatter.cpp

```diff
@@ -45,15 +45,15 @@ class BeatsFormatter final :
       // [0, range), so add 1.
 
       auto& beatsField = mFields.emplace_back(
-         NumericField { std::max<size_t>(12, mUpperTimeSignature + 1), true });
+         NumericField::ForRange(std::max(11, mUpperTimeSignature + 1)));
```

Comment:
Is the change from 12 to 11 at 50d15a573ca3fc478b09ef94a9009b22c099cd76 intended?